dashboard/dashboard.py

```diff
@@ -112,7 +112,24 @@ def validate_phone_number(phone_number: str, region='GB') -> bool:
     })
     zoom_level = 1.5
     st.map(iss_df, zoom=zoom_level)
-
+    st.write('Data sources:')
```

Comment:
Nice images for each data source with links to documentation. Shows users exactly where the data is from.